pkg/controller/user/user_controller.go

```diff
@@ -205,17 +203,41 @@ func (r *ReconcileUser) assertExistingUser(ctx context.Context, harborClient *h.
 	if err != nil {
 		return err
 	}
+
 	pw, err := helper.GetValueFromSecret(sec, "password")
 	if err != nil {
 		return err
 	}
+	pwHash, err := helper.GenerateHashFromInterfaces([]interface{}{pw})
+	if err != nil {
+		return err
+	}
 
 	heldUser, err := internal.GetUser(user, harborClient)
 	if err != nil && err != internal.ErrUserNotFound {
 		return err
 	}
+
+	patch := client.MergeFrom(user.DeepCopy())
 	if err == internal.ErrUserNotFound {
-		return r.createUser(harborClient, user, pw)
+		user.Status.PasswordHash = pwHash.Short()
+		fmt.Println("Password:", pw)
```

Comment:
Guess this is a left-over debug output?

Reply:
good find! And yes, it is a left-over debug output!